docs/dev/project_management.rst

```diff
@@ -8,16 +8,72 @@ management tools as well as `Zenhub <https://www.zenhub.com>`__ which provides
 additional features for sprint planning, task estimation, and progress reports.
 
 -------------------------------------------------------------------------------
-Issues and Project Tracking
+Agile Scrum Kanban Iteration SDLC Process
 -------------------------------------------------------------------------------
-We use `Github issues <https://github.com/catalyst-cooperative/pudl/issues>`__ to
-track bugs, enhancements, support requests, and just about any other work that goes
-into the project. Try to make sure that issues have informative tags so we can find
-them easily.
-
-We use Zenhub Sprints, Epics, and Releases to track our progress. These won't be
-visible unless you have the `ZenHub browser extension
-<https://www.zenhub.com/extension>`__ installed.
+
+We merge code to ``dev`` continuously, but plan work in two-week sprints.
+
+We track work as items (GitHub issues and PRs) in a Catalyst-wide Github Project.
+
+During each sprint, people on each team pull from a list of prioritized & scoped
+items.
+
+Each team has its own independent sprint process.
+
+We have 2 or 3 meetings per sprint:
+
+- sprint planning: overall goal is "get a good idea of what work is coming up
+  in the sprint"
+
+  - mark items with a new sprint
+
+    - enough that that we feel like there's enough work to do
+
+    - few enough that we don't feel overwhelmed
+
+- (Inframundo only - other teams do this asynchronously): mid-sprint check-in:
```

Comment:
I think this can also be a useful time to check-in if anything high priority is stuck for some reason